flavors/posture.go

```diff
@@ -91,15 +99,13 @@ func NewPosture(_ *beat.Beat, cfg *agentconfig.C) (*posture, error) {
 	// namespace will be passed as param from fleet on https://github.com/elastic/security-team/issues/2383 and it's user configurable
 	resultsIndex := config.Datastream("", config.ResultsDatastreamIndexPrefix)
 
-	commonDataProvider := dataprovider.NewCommonDataProvider(log, c)
-	commonData, err := commonDataProvider.FetchCommonData(ctx)
+	cdp, err := GetCommonDataProvider(ctx, log, *c)
```

Comment:
opinion: I know it's a common practice in go, but I think too many acronyms can hurt the code readability

Reply:
I agree in general, here the use of the variable is very close to the declaration
https://dave.cheney.net/practical-go/presentations/gophercon-singapore-2019.html#_identifier_length